dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/ValidatorUtils.java

```diff
@@ -75,16 +75,13 @@ public static boolean flowsIsSet(OAuthFlows flows) {
             OAuthFlow password = flows.getPassword();
             if (implicit != null) {
                 if (StringUtils.isNotBlank(implicit.getAuthorizationUrl()) ||
-                    StringUtils.isNotBlank(implicit.getTokenUrl()) ||
```

Comment:
I believe all of these checks are still required, since setting one of the value (even if the field is not applicable) implies that the user unnecessarily defined an OAuth Flow.